xmovie/core.py

```diff
@@ -253,6 +254,7 @@ def __init__(
         pixelwidth=1920,
         pixelheight=1080,
         dpi=200,
+        bbox_inches=None,
```

Comment:
We might add other kwargs for `.savefig()` here (see comments below). If so should we have some dict that catches all `save_kwargs=None` (we can then set defaults we like in the init)? Otherwise we might be accumulating a bunch of input arguments and the signature of this might become unruly?